docs/sources/reference/components/otelcol.exporter.kafka.md

```diff
@@ -0,0 +1,244 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol.exporter.kafka/
+description: Learn about otelcol.exporter.kafka
+title: otelcol.exporter.kafka
+---
+
+# otelcol.exporter.kafka
+
+`otelcol.exporter.kafka` accepts logs, metrics, and traces telemetry data from 
+other `otelcol` components and sends it to Kafka.
+
+It is important to use `otelcol.exporter.kafka` together with `otelcol.processor.batch`
+in order to make sure `otelcol.exporter.kafka` doesn't slow down due to sending Kafka a huge number of small payloads.
+
+> **NOTE**: `otelcol.exporter.kafka` is a wrapper over the upstream
+> OpenTelemetry Collector `kafka` exporter from the `otelcol-contrib`
+> distribution. Bug reports or feature requests will be redirected to the
+> upstream repository, if necessary.
+
+Multiple `otelcol.exporter.kafka` components can be specified by giving them
```

Comment:
This (and other parts of the standard text) is all passive voice. I'm leaving it as-is here... but at some point we should update the template or boilerplate we use here to active voice.